src/app/about/page.jsx

```diff
@@ -1,32 +1,31 @@
 import imgTop from "@/assets/about/top.svg";
 import imgBot from "@/assets/about/bot.svg";
-import Logo from "@/assets/Logosimple.svg";
+import Logo from "@/assets/about/Logo.png";
 import ImgAbout from "./imgAbout";
 import Image from "next/image";
 import text from "./text";
 
 export default function About() {
   return (
-    <div className="w-full min-h-screen relative">
+    <div className="w-full">
       <ImgAbout image={imgTop} />
-      <div className="w-full h-full flex flex-col px-56 ">
-        <h1 className=" text-2xl pb-10">
-          Quienes <span className="text-[#3DAADD]">somos</span>
+      <div className="w-full flex flex-col px-10 md:px-16 lg:px-52">
+        <h1 className="text-2xl pb-5 lg:pb-10">
+          Quiénes <span className="text-[#3DAADD]">somos</span>
         </h1>
-        <div className="pb-14 ">
+        <div className="pb-14">
           {text.map((item, index) => (
-            <p key={index} className="text-xs pb-4 text-gray-950">
+            <p key={index} className="text-sm pb-4 text-gray-950">
               {item}
             </p>
           ))}
-          ;
         </div>
-        <p className="text-sm text-center">
+        <p className="text-base text-center">
           ¡Bienvenidos a FisiomFulness, donde el bienestar es nuestro compromiso
           y su satisfacción es nuestro éxito!
         </p>
-        <div className="w-full h-full flex justify-center item-center py-10">
-          <Image src={Logo} alt="Logo" width={30} height={30} />
+        <div className="w-full flex justify-center item-center pt-16">
+          <Image src={Logo} width={30} height={30} alt="No"></Image>
```

Comment:
En la carpeta _public_ se encuentran los logos, es mejor traerlos desde ahi y eliminar _src/assets/about/Logo.png_.
```suggestion
          <Image src="/logo_simple.webp" width={30} height={30} alt="No"></Image>
```